docs/product-release-notes/src/main/asciidoc/con-bxms-rn-whats-new.adoc

```diff
@@ -0,0 +1,156 @@
+ 
+=== Business Central Monitoring web application
+A new distribution `war` file that enables you to build custom cloud images has been added in this release.The Business Central Monitoring web application should be used with Decision Server instances to manage all run time capabilities in the cloud. This includes managing containers, process instances, tasks, and dashboards. Unlike the standard Business Central distribution, no authoring capabilities are offered.
+
+NOTE:
+For on premise deployments, use the standard Business Central `war` file.
+ 
+=== Work Item archetype
+
+To help users build custom service tasks (work items), {PRODUCT} comes with the Work Item archetype that generates the majority of items required to build a custom service task.  It does include:
+
+* A Work Item Definition (WID) file
+* The Work Item Handler implementation class
+* The Work Item Handler test class
+
+The Maven assembly (zip) packages everything at build time so it can be consumed by the Service Repository and therefore be used from within the Web Designer.
+
+=== KIE Server Maven plugins
+KIE Server has been enhanced with additional Maven plugins that enable interaction with the KIE Server and KIE Controller REST API directly from within a build. This facilitates easier integration with CI/CD pipelines when building KJars so they can be directly deployed to the execution environment (both managed and unmanaged KIE Servers).
+
+== Business Resource Planner
+* Multi-threaded partitioned search is available. Business Resource Planner now has out-of-the box support for solving a single data set by partitioning across multiple threads. This enhancement makes use of multiple CPU cores for a single problem. Partitioned search can implement geo-fencing for Vehicle Routing use cases. 
+* The Solution interface is deprecated. Your solution class requires only the `@PlanningSolution` annotation.
+* You no longer need to define the score type in the solver configuration. Business Resource Planner now picks it up automatically from the domain model.
+* The Business Resource Planner Solver editor screen now supports adding all termination types, including composite termination. The phase configuration section enables you to tweak Construction Heuristic settings and select Local Search algorithm to optimize your planning problem.
+* Examples are now part of Business Central and they work offline as well. Leverage tag-based filtering to quickly access projects from the field you are interested in.
+* The Business Resource Planner domain editor can now specify a planning entity difficulty. Navigate through the object hierarchy and define the sorting attributes. Several construction heuristic algorithms use this information to construct a better initial solution.
```

Comment:
I am not sure how "Navigate through the object hierarchy and define the sorting attributes" works here. It sounds like a piece of a procedure. It is not obviously connected to the "planning entity difficulty" thing. Perhaps the difficulty is one of the sorting attributes? Then suggestion "To set the difficulty for a planning entity, find the entity in the object hierarchy and open its sorting attributes" (but there is a huge amount of guesswork in this suggestion, please double-check)